### DISCUSSION TITLE
是否原生支持RPC等协议

### DISCUSSION THREAD

**Discussion by imath60** [httprunner/httprunner#1179] [2 - 答疑（Q&A）]
除HTTP协议外 是否考虑支持其他常用协议如DUBBO GRPC WEBRTC等

**Comment by debugtalk** [comment]
@imath60 规划会支持更多协议，但近期重点还在 HTTP 方面

**Reply by JolieYang** [reply]
官网多协议扩展-RPC页面是空白的，不知道现在支持RPC协议了吗？
